crates/core/src/surfnet/svm.rs

```diff
@@ -1207,6 +1202,38 @@ impl SurfnetSvm {
         self.slot_subscriptions
             .retain(|tx| tx.send(SlotInfo { slot, parent, root }).is_ok());
     }
+
+    pub fn subscribe_for_logs_updates(
```

Comment:
We may have already set a precedence for this with the other WS methods, idk, but are we ever removing from the list of `log_subscriptions` here? I know a user can unsubscribe and the WS channel is disconnected, but do we clean up this list when that happens?

Reply:
https://github.com/txtx/surfpool/issues/247